stories/Who_was_protected_by_the_first_COVID-19_lockdown.ipynb

```diff
@@ -0,0 +1,2412 @@
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>95500</th>\n",
+       "      <td>E07000033</td>\n",
+       "      <td>Local Authority</td>\n",
+       "      <td>Bolsover</td>\n",
+       "      <td>All causes</td>\n",
+       "      <td>Hospital</td>\n",
+       "      <td>2</td>\n",
+       "      <td>2020-06-10</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>78938</th>\n",
+       "      <td>E07000012</td>\n",
+       "      <td>Local Authority</td>\n",
+       "      <td>South Cambridgeshire</td>\n",
+       "      <td>All causes</td>\n",
+       "      <td>Home</td>\n",
+       "      <td>11</td>\n",
+       "      <td>2020-05-13</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>12416</th>\n",
+       "      <td>E06000006</td>\n",
+       "      <td>Local Authority</td>\n",
+       "      <td>Halton</td>\n",
+       "      <td>COVID 19</td>\n",
+       "      <td>Home</td>\n",
+       "      <td>0</td>\n",
+       "      <td>2020-01-22</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "       Area code   Geography type             Area name Cause of death  \\\n",
+       "57910  E06000024  Local Authority        North Somerset       COVID 19   \n",
+       "57805  E06000016  Local Authority             Leicester     All causes   \n",
+       "95500  E07000033  Local Authority              Bolsover     All causes   \n",
+       "78938  E07000012  Local Authority  South Cambridgeshire     All causes   \n",
+       "12416  E06000006  Local Authority                Halton       COVID 19   \n",
+       "\n",
+       "      Place of death  Number of deaths        Date  \n",
+       "57910       Hospital                 5  2020-04-08  \n",
+       "57805      Elsewhere                 0  2020-04-08  \n",
+       "95500       Hospital                 2  2020-06-10  \n",
+       "78938           Home                11  2020-05-13  \n",
+       "12416           Home                 0  2020-01-22  "
+      ]
+     },
+     "execution_count": 23,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "mortality_df.sample(n=5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you want to reproduce the results from the initial [ONS report](https://www.ons.gov.uk/peoplepopulationandcommunity/birthsdeathsandmarriages/deaths/bulletins/deathsinvolvingcovid19bylocalareasanddeprivation/latest), you can restrict the date ranges of the data by uncommenting these lines."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2020-10-07T15:26:49.151374Z",
+     "start_time": "2020-10-07T15:26:49.149766Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "#mortality_df = mortality_df[mortality_df['Date'] > '2020-03-01']\n",
```

Comment:
If these lines are uncommented, the line
```
third_df = filter_date_and_aggregate(mortality_sum_df, third_date_range)
```
fails because there aren't any dates available in the selected time period. It all makes sense after a look at the code, but is there a check you could do to make sure that it fails more elegantly rather than with a big error message?